apps/scale.go

```diff
@@ -107,6 +109,13 @@ func Scale(app string, sync bool, types []string) error {
 	res, err := c.AppsScale(app, scaleParams)
 	if err != nil {
 		if !utils.IsPaymentRequiredAndFreeTrialExceededError(err) {
+			reqestFailedError, _ := errors.ErrgoRoot(err).(*http.RequestFailedError)
```

Comment:
```suggestion
			reqestFailedError, ok := errors.ErrgoRoot(err).(*http.RequestFailedError)
```

Shouldn't you test the `ok` value? What happens if the error is not of type `http.RequestFailedError`? :thinking: